plugins/push/frontend/public/javascripts/countly.models.js

```diff
@@ -192,12 +200,80 @@
                     sent: pushNotificationDtoItem.result.sent,
                     actioned: pushNotificationDtoItem.result.actioned,
                     createdBy: pushNotificationDtoItem.creator,
-                    platform: self.mapPlatforms(pushNotificationDtoItem.platforms),
+                    platforms: self.mapPlatforms(pushNotificationDtoItem.platforms),
                     content: pushNotificationDtoItem.messagePerLocale.default,
                 };
             });
             return rowsModel;
         },
+        mapMessageButtons: function(numberOfButtons, dto) {
+            if (numberOfButtons === 1) {
+                return [{label: dto.messagePerLocale["default|0|t"], url: dto.messagePerLocale["default|0|l"]}];
+            }
+            if (numberOfButtons === 2) {
+                return [
+                    {label: dto.messagePerLocale["default|0|t"], url: dto.messagePerLocale["default|0|l"]},
+                    {label: dto.messagePerLocale["default|1|t"], url: dto.messagePerLocale["default|1|l"]},
```

Comment:
Just out of curiosity... How does this "default|0|t" notation work? Cartesian product of some variables may be?

Reply:
I am not sure. I think @iartem knows the answer to your question.